compiler/onecc-docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:18.04 
+
+RUN apt-get update 
+
+RUN apt-get install -y wget 
+
+RUN wget https://github.com/Samsung/ONE/releases/download/1.21.0/one-compiler_1.21.0_amd64.deb 
+
+RUN apt install -y ./one-compiler_1.21.0_amd64.deb 
```

Comment:
I think `apt-get` would be better to use in bash script file. Because the `apt` is designed to Linux user and does not have a stable CLI interface.

Reply:
@jyoungyun
Do you mean to use the apt-get option rather than the wget option? Or do you mean to change the apt of the Debian package installation part?

I wonder if this is in line with your intentions
```
FROM ubuntu:18.04 

RUN apt-get update && apt-get install -y wget

RUN wget https://github.com/Samsung/ONE/releases/download/1.21.0/one-compiler_1.21.0_amd64.deb 
RUN apt-get install -y ./one-compiler_1.21.0_amd64.deb 
```

Reply:
AFAIK, `apt` is not appropriate to use in bash script. :) I suggest to use `apt-get` instead of `apt`.